presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/TestRemoveDistinctFromSemiJoin.java

```diff
@@ -42,11 +47,23 @@ public void test()
                     Symbol filteringSourceKey = p.symbol("custkey_1", BigintType.BIGINT);
                     Symbol outputKey = p.symbol("orderkey", BigintType.BIGINT);
                     return p.semiJoin(
-                            p.tableScan("orders", ImmutableMap.of("custkey", "custkey")),
+                            p.tableScan(
+                                    new TableHandle(
```

Comment:
maybe plan builder could have a method, `p.tableHandle("local", "local", "orders", TINY_SCALE_FACTOR)` where you could create transactions